app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -28,12 +28,18 @@ def pretty_print(classes)
       if (mod = klass.gsub(/::.*/, ""))
         hash[mod] ||= []
         hash[mod] << klass
-      else
-        next
       end
     end
     hash.keys.sort.map do |key|
-      link_to key,{}, {:remote => true, :rel => "popover", :data => {"content" => hash[key].sort.join('<br>').html_safe, "original-title" => key}}
+      link_to key, {}, { :remote => true,
+                         :rel => "popover",
+                         :data => {
+                           "content" => hash[key].sort.join('<br>').html_safe,
+                           "original-title" => "#{key} has <span class='label label-info'>#{hash[key].size}</span> classes",
```

Comment:
1. does not work for singular. e.g. X has one class.
2. strings needs to be extracted